core/template.js

```diff
@@ -990,6 +990,26 @@ var Template = Montage.specialize( /** @lends Template# */ {
                     argumentElement = templateArgumentProvider.getTemplateArgumentElement(
                         parameterName);
 
+                    if (parameterName === "*" && parameterElement.childElementCount > 0) { // has default
+                        if (argumentElement.childElementCount > 0 ||
+                            !!(argumentElement.firstChild &&
+                                argumentElement.firstChild.data &&
+                                argumentElement.firstChild.data.trim())
+                        ) { // has arguments
+                            var elementIds = this._getChildrenElementIds(parameterElement);
+                            
+                            for (var i = 0, length = elementIds.length; i < length; i++) {
+                                serialization.removeObject(elementIds[0]);
+                            }
+                        } else {
+                            var range = parameterElement.ownerDocument.createRange();
```

Comment:
I think Range can be reused, might be worth looking into that to minimize garbage creation